inference-engine/thirdparty/clDNN/kernel_selector/common/common_types.h

```diff
@@ -73,7 +73,8 @@ enum class KernelType {
     LOOP,
     NON_MAX_SUPPRESSION,
     DETECTION_OUTPUT,
-    EXPERIMENTAL_DETECTRON_ROI_FEATURE_EXTRACTOR
+    EXPERIMENTAL_DETECTRON_ROI_FEATURE_EXTRACTOR,
+    RANDOM_UNIFORM
```

Comment:
yes, I thought about it and it would be reasonable to add it alphabetically in case it was already sorted. But now there is no sense to do it. Much better to make separate PR with fully sorted list